app/forms/network-interface-edit.tsx

```diff
@@ -57,12 +58,11 @@ export function EditNetworkInterfaceForm({
 
   const transitIpsForm = useForm({ defaultValues: { transitIp: '' } })
 
-  const submitTransitIp = () => {
-    const transitIp = transitIpsForm.getValues('transitIp')
+  const submitTransitIp = transitIpsForm.handleSubmit(({ transitIp }) => {
     if (!transitIp) return
     form.setValue('transitIps', [...transitIps, transitIp])
     transitIpsForm.reset()
-  }
+  })
```

Comment:
Without wrapping this in `handleSubmit`, it doesn't run the `validate` function on submit. This way the inner function never gets called unless validation passes.